includes/classes/Command.php

```diff
@@ -168,8 +179,10 @@ function( $post_type ) use ( $post_types ) {
 			WP_CLI::line( 'Preparing for the next batch...' );
 			sleep( 3 );
 
-			// Free up memory.
-			$this->stop_the_insanity();
+			if ( class_exists( 'WPCOM_VIP_CLI_Command' ) ) {
+				// Free up memory.
+				$this->stop_the_insanity();
```

Comment:
I wonder if it would be worth adding our own version of this method, so if someone isn't on VIP, they still get the performance benefits this provides? I'm not worried about the `start_bulk_operation` or `end_bulk_operation` methods, as those don't do much in our case but having this method could help some with performance when processing large batches of content